packages/foreman/rubygem-net-scp/rubygem-net-scp.spec

```diff
@@ -1,53 +1,43 @@
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name net-scp
 
+Name: rubygem-%{gem_name}
+Version: 3.0.0
```

Comment:
1.2.1 to 3.0.0 is a big change. Needs some further analysis

Reply:
Looks like this isn't a major bump in the semver sense. It just raises the upper bound of net-ssh.